### ISSUE TITLE
which classes need to modify to add new col in database?

### ISSUE THREAD

**Issue by cispune** [tananaev/traccar#1609]
need to add some col in position table, which files to modify

**Comment by tananaev** [comment]
You can easily do it with minor modification to `DataManager` class.

**Comment by cispune** [comment]
i need to add ignition parameter in separate col is it possible?

On Sat, Dec 19, 2015 at 4:02 AM, Anton Tananaev <notifications@github.com>
wrote:

> You can easily do it with minor modification to DataManager class.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tananaev/traccar/issues/1609#issuecomment-165912120>.
>



-- 
CIS Technologies (P) Ltd.

**Comment by tananaev** [comment]
Yes, it is possible. All the SQL queries are in the config file, so you can do anything you want.
